en/ros/ros1_via_ros2.md

```diff
@@ -20,14 +20,14 @@ The PX4 development team recommend that all users [upgrade to ROS 2](../ros/ros2
 
 ## Overview
 
-The application pipeline for ROS (1) bridged over ROS 2 is shown below.
+The application pipeline for ROS (1) bridged over ROS 2 is shown below. 
```

Comment:
I've "touched" this doc just to remind up it will need to be updated to explain how to do ROS 1 over ROS 2 with new architecture(s)